scripts/docker-helper/readme.md

```diff
@@ -177,3 +185,72 @@ If you REALLY get stuck and want to destroy _**ALL**_ docker containers/volumes/
 ```
 docker stop $(docker ps -q)&&docker system prune&&docker volume prune
 ```
+
+## cht-docker-compose.log
+
+This log will be output every time you call the script.  It will be created and appended to in the directory where you specified your environment file to be (`-e PATH/TO/env_file`). 
```

Comment:
Minor wording suggestion:
```suggestion
This log will be output every time you call the script.  It will be created and appended to in the directory where you specified your environment file (`-e PATH/TO/env_file`). 
```